src/components/ArcGISWebMap.tsx

```diff
@@ -710,6 +756,7 @@ function FormElementArcGISWebMap({
         id={id}
         aria-describedby={props['aria-describedby']}
       />
+      <img src={_screenshot} alt="Screenshot" />
```

Comment:
Is this just for POC to see it working?

Reply:
Oh yeah, not actually part of the solution